lib/google_drive/collection.rb

```diff
@@ -35,19 +35,24 @@ def remove(file)
     # Creates a sub-folder with given title. Returns GoogleDrive::Collection
     # object.
     def create_subcollection(title)
+      create_file(mime_type: 'application/vnd.google-apps.folder')
+    end
+
+    alias create_subfolder create_subcollection
+
+    def create_file(title, params = {})
```

Comment:
Thanks, looks useful, but can you define `create_spreadsheet` instead, to be consistent with a method in GoogleDrive::Session?

I believe `create_*` method is particularly useful for a spreadsheet and a folder. For other types, people would probably want to use `upload_*` methods instead.

You can still define `create_file` to reuse implementation, but probably as a private method.

Reply:
The problem with defining `create_spreadsheet` is that we then have to define `create_x` for every possible mime type. Maybe we can define `create_spreadsheet` and still make `create_file` a public function?

Reply:
As mentioned above, "creating an empty file" is not very useful for many of other file types (e.g., image/png). That's why I suggested to only add `create_spreadsheet`.

But if you think it's useful (it may be for document and presentation for example), it's OK to leave it public, but can you:

* add GoogleDrive::Session#create_file as well (for consistency)
* add GoogleDrive::Collection#create_spreadsheet as well (for consistency)
* add document comment for create_file, optionally with a link to a document of available mime types